### PR TITLE
Gene search AF cutoff

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: ui/shared/components/panel/search/GeneBreakdown.jsx [hunk 1]
```diff
@@ -39,13 +39,6 @@ const COLUMNS = [
     format: row => (
       <SearchResultsLink
         location={row.geneId}
-        projectFamilies={Object.entries(row.families.reduce((acc, { family }) => {
-          if (!acc[family.projectGuid]) {
-            acc[family.projectGuid] = []
-          }
-          acc[family.projectGuid].push(family.familyGuid)
-          return acc
-        }, {})).map(([projectGuid, familyGuids]) => ({ projectGuid, familyGuids }))}
```
Comment (ShifaSZ) [inline ui/shared/components/panel/search/GeneBreakdown.jsx:48]: Is there any reason that this parameter can be eliminated?

File: ui/shared/components/panel/variants/VariantGene.jsx [hunk 2]
```diff
@@ -350,6 +350,16 @@ GeneDetails.propTypes = {
   rnaSeqData: PropTypes.object,
 }
 
+const GeneSearchLinkWithPopup = props => (
+  <Popup
+    trigger={
+      <GeneSearchLink {...props} />
+    }
+    content="Search for all variants with AF < 10% in this gene present in any affected individual"
```
Comment (ShifaSZ) [inline ui/shared/components/panel/variants/VariantGene.jsx:358]: `AF < 10%` is just the default value. It is not true after user changes it. Add `by default` to the sentence?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: ui/shared/components/panel/search/GeneBreakdown.jsx (hunk 1 above)
Comment (hanars) [reply ui/shared/components/panel/search/GeneBreakdown.jsx:48]: yes it was not being used at all, and when I was trying to figure out how the component was working its presence was super confusing

File: ui/shared/components/panel/variants/VariantGene.jsx (hunk 2 above)
Comment (hanars) [reply ui/shared/components/panel/variants/VariantGene.jsx:358]: it is going to open up a search page with search results with those exact filters applied, its not pre-populating a form you can adjust. Also this was explicitly asked for in the ticket

Comment (ShifaSZ) [reply ui/shared/components/panel/variants/VariantGene.jsx:358]: I see.

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**